File: Napakalaki_Java/src/napakalaki/Cultist.java
/**
 *  Marco Manuel Fernández Pranno, 2015 (Granada)
 *  Para la asignatura PDOO (Universidad de Granada)
 *  Licencia GPLv3 a partir del 15/02/16, hasta entonces todos los derechos reservados.
 */
package napakalaki;

public class Cultist {
    private int gainedLevels;
    private String name;
    
    public Cultist(String name, int gainedLevels){
        this.name = name;
        this.gainedLevels = gainedLevels;
    }
    
    public int getGainedLevels(){
        return gainedLevels;
    }
}

File: Napakalaki_Ruby/pkg/Napakalaki_Ruby-0.0.1/lib/cultist.rb
#encoding: utf-8

=begin
  Marco Manuel Fernández Pranno, 2015 (Granada)
  Para la asignatura PDOO (Universidad de Granada)
  Licencia GPLv3 a partir del 15/02/16, hasta entonces todos los derechos reservados.
=end 

class Cultist
  
  def initialize(name,gainedLevels)
    @name = name
    @gainedLevels = gainedLevels
  end
  
  attr_reader :gainedLevels
end